spec/features/projects_spec.rb

```diff
@@ -45,6 +45,28 @@
           expect(user.projects.count).to eq(0)
         end
       end
+
+      describe 'archivement' do
+        before { visit edit_project_path(project) }
+
+        context 'with a non archived project' do
+          let(:project) { create :project, users: [user], teams: [current_team] }
+
+          scenario 'there is no button' do
+            expect(page).not_to have_selector(:link_or_button, 'Archive')
+            expect(page).not_to have_selector(:link_or_button, 'Unarchive')
+          end
+        end
+
+        context 'with an archived project' do
+          let(:project) { create :project, :archived, users: [user], teams: [current_team] }
+
+          scenario 'there is no button' do
+            expect(page).not_to have_selector(:link_or_button, 'Archive')
```

Comment:
Same here. In this case the project is archived, shouldn't the unarchive button be enabled?

Reply:
Same here, but the opposite, the if it's archived there is an unarchive button, but there isn't an archive one.

I think these tests are poorly worded.